api/models/photo.py

```diff
@@ -392,6 +397,15 @@ def _extract_date_time_from_exif(self, commit=True):
             except Exception:
                 timestamp_from_exif = None
 
+        if gps_time:
```

Comment:
Doesn't look like a fallback to me - shouldn't it be under `if timestamp_from_exif is None and gps_time:` ?

Reply:
Also GPS Date/Time seems to be storing UTC time (at least for examples I looked at) - so that would likely benefit from timezonefinder for making potentially better guess at local time (similar to video creation time).